project/plugins.sbt

```diff
@@ -1,3 +1,3 @@
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "2.1.4")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "2.3.1")
 addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "3.10.0")
```

Comment:
Shouldn't we update to the latest version, to be able to publish using new Sonatype API? sbt-pgp Also has a newer version.

Reply:
For now snapshot publication works with little hack to properly handle creds in SONATYPE_ envs.
I'll check whether those versions works during full-release after the PR.